Header header
float32 posx
float32 posy
float32 posz
float32 veltx
float32 velty
float32 veltz
float32 filterveltx
float32 filtervelty
float32 filterveltz
float32 pendulum_posx
float32 pendulum_posy
float32 pendulum_posz
float32 pendulum_attx
float32 pendulum_atty 
float32 roll
float32 yaw
float32 pitch
#data health
uint8 health_flag
uint8 network_health_flag
uint8 pendulum_health_flag
float32 delay
uint8 lost_data